docker-compose-solo.yml

```diff
@@ -30,17 +30,14 @@ services:
       - "8080:8080"
```

Comment:
I have trouble with this port being opened by default, because it often conflicts with other open projects.
I prefer having the port allocating in the .override file
Unfortunately .override does not allow deactivating ports
But OK, I can always comment this line